packages/patternfly-4/react-core/src/components/Chip/Chip.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import ChipButton from './ChipButton';
+import { Tooltip, TooltipPosition } from '../Tooltip';
+import { TimesCircleIcon } from '@patternfly/react-icons';
+import styles from '@patternfly/patternfly-next/components/Chip/chip.css';
+import GenerateId from '../../internal/GenerateId/GenerateId';
+
+class Chip extends React.Component {
+  renderOverflowChip = () => {
+    const { children, className, onClick } = this.props;
+    return (
+      <div className={css(styles.chip, styles.modifiers.overflow, className)}>
+        <ChipButton onClick={onClick} aria-label="Expand Chip">
```

Comment:
Don't need the aria-label here (from what i can see by looking at the Core docs)

Reply:
Just had a convo about this with @tlabaj 

"I think what you have actually correct.  I think the example in core should actually have a aria-label.  PF-Next calls for one whenever the "plain" modifier is used for a button."

Reply:
Should we change the button component to reflect what is in core? As it currently stands, there is a dependency on the aria-label if it is a plain button.

Reply:
I talked with @jgiardino and @mcoker.  The pf-m-plain documentation will be updated in core.  That modifier can be used for buttons that contain text. If the button contains text (like the chip here), the aria label is not needed.  I have opened an issue #1184  to update the button component.